.github/workflows/wheel_noarch.yml

```diff
@@ -41,72 +41,67 @@ jobs:
 
     steps:
       - name: Checkout PennyLane-Lightning
+        if: ${{ matrix.pl_backend == 'lightning_qubit'}}
         uses: actions/checkout@v4
-        with:
-          path: main
+
 
       - uses: actions/setup-python@v5
+        if: ${{ matrix.pl_backend == 'lightning_qubit'}}
         with:
           python-version: '3.9'
 
       - name: Upgrade pip
+        if: ${{ matrix.pl_backend == 'lightning_qubit'}}
         run: |
           python -m pip install --upgrade pip
 
-      - name: Install CMake and ninja
+      - name: Install dependencies
+        if: ${{ matrix.pl_backend == 'lightning_qubit'}}
         run: |
-          python -m pip install --upgrade cmake ninja
+          python -m pip install --upgrade cmake ninja toml
```

Comment:
Do we need these dep stages now that we have them spec'd in the TOML build section?

Reply:
Good point. I don't think we need it. Let me test it.